ClosedXML_Examples/ConditionalFormatting/ConditionalFormatting.cs

```diff
@@ -634,4 +634,30 @@ public void Create(String filePath)
             workbook.SaveAs(filePath);
         }
     }
+
+    public class CFStopIfTrue : IXLExample
+    {
+        public void Create(String filePath)
+        {
+            var workbook = new XLWorkbook();
+            var ws = workbook.AddWorksheet("Sheet1");
+
+            ws.FirstCell().SetValue(6)
+                .CellBelow().SetValue(1)
+                .CellBelow().SetValue(2)
+                .CellBelow().SetValue(3);
+
+            var cf = ws.RangeUsed().AddConditionalFormat();

```

Comment:
It should be noted that percent and bottom are not exposed this way, but I can make the change requested.